charts/openfaas-cloud/templates/edge-auth-dep.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.edgeAuth.name }}
+  labels:
+    app.kubernetes.io/name: openfaas-cloud
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/component: {{ .Values.edgeAuth.name }}
+spec:
+  replicas: {{ .Values.edgeAuth.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ .Values.edgeAuth.name }}
+  template:
+    metadata:
+      annotations:
+        prometheus.io.scrape: {{ .Values.global.prometheusScrape | quote }}
+      labels:
+        app: {{ .Values.edgeAuth.name }}
+    spec:
+      volumes:
+        - name: jwt-private-key
```

Comment:
Do we ever template these values or names? (if not, that's fine)

Reply:
this is from the README for the edge-auth component

```sh
kubectl -n openfaas create secret generic jwt-private-key --from-file=./key
kubectl -n openfaas create secret generic jwt-public-key --from-file=./key.pub
```

then we use these named secrets, ofc-bootstrap also uses these names, so I think for now we dont need a template variable.